### PR TITLE
more funny for intercom

### PR DESCRIPTION
**Pull request by zhaguanyang**
Added ``Intercom.RemainingCooldown`` for getting or setting the intercom's cooldown.
Added ``Intercom.Reset()`` to reset the intercom's cooldown to zero.
Added ``Intercom.Timeout()`` to timeout the intercom

### CODE REVIEW

**Comment by zhaguanyang** [comment]
image
Tested with above commands

File: Exiled.API/Features/Intercom.cs [hunk 1]
```diff
@@ -55,5 +64,21 @@ public static float SpeechRemainingTime
...
+        /// Reset the intercom's cooldown.
+        /// </summary>
+        public static void Reset() => RemainingCooldown = -1f;
+
+        /// <summary>
+        /// Times out the intercom.
+        /// </summary>
+        public static void Timeout()
+        {
+            if(InUse)
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Intercom.cs:78]: Leave a space after the `if` statement like: `if (InUse)`[...]

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Intercom.cs (hunk 1 above)
Comment (zhaguanyang) [reply Exiled.API/Features/Intercom.cs]: done

**Review by xRoier — COMMENTED** [review]
LGTM

**Pull request merged**